how-to/containerd-kata.md

```diff
@@ -0,0 +1,345 @@
+# How to use Kata Containers and Containerd
+
+- [Concepts](#concepts)
+  - [Kubernetes RuntimeClass](#kubernetes-runtimeclass)
+  - [Containerd Runtime V2 API (Shim V2 API)](#containerd-runtime-v2-api-(shim-v2-api))
+- [Install](#install)
+  - [Install Kata Containers](#install-kata-containers)
+  - [Install containerd with cri plugin](#install-containerd-with-cri-plugin)
+  - [Install CNI plugins](#install-cni-plugins)
+  - [Install cri-tools](#install-cri-tools)
+- [Configuration](#configuration)
+  - [Configure containerd to use Kata Containers](#configure-containerd-to-use-kata-containers)
+  - [Configuration for cri-tools](#configuration-for-cri-tools)
+- [Run](#run)
+  - [Launch containers with ctr command line](#launch-containers-with-ctr-command-line)
+  - [Launch Pods with crictl command line](#launch-pods-with-crictl-command-line)
+
+This document covers the installation and configuration of [containerd](https://containerd.io/) 
+and [Kata Containers](https://katacontainers.io). The containerd will provide not only the `ctr`
+command line tool, but also the [CRI](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/) 
+interface for [Kubernetes](https://kubernetes.io) and other CRI clients.
+
+This document is mainly written for Kata Containers v1.5.0-rc2 or above and containerd v1.2.0 or above.
+The document addressed the previous versions as well though we suggest users upgrade to the newer versions for better support.
+
+## Concepts
+
+### Kubernetes RuntimeClass
```

Comment:
I think runtimeclass concept should be in a different document, as this is specific to k8s, not containerd

Reply:
I agree `RuntimeClass` is not a containerd specific concept, but we need some background to introduce the containerd configuration.

I don't think we should have a full document on runtime class. As a result, I put a paragraph here for information.

I plan to
- move cri-o related doc contents to how-to, and update to the latest release (#333 )
- update the kubernetes howto to support both cri-o and containerd, and move the runtime class back to that document

What's your opinion, @egernst?